### ISSUE TITLE
[BUG] Example js in accessibility-checker-engine readme is incorrect

### ISSUE THREAD

**Issue by hadleyn** [IBMa/equal-access#123]
## What tool(s) are you using?

- [x ] `accessibility-checker-engine`
- [ ] `accessibility-checker`
- [ ] `karma-accessibility-checker`
- [ ] `rule-server`
- [ ] `boilerplates`
- [ ] `other`

## Detailed description

> Node version:
> Platform: 

> Describe in detail the issue you're having.

## Steps to reproduce the issue

The javascript code in the example on the `README.md` for accessibility-checker-engine is incorrect. It's missing an `(async() {` wrap and the `const browser` is set twice. Both situations cause errors that prevent the example from running.

**Comment by tombrunet** [comment]
Closed by https://github.com/IBMa/equal-access/pull/124

Thanks!
